### ISSUE TITLE
Invalid css format when using calc()

### ISSUE THREAD

**Issue by goras** [mpaupulaire4/bulma-styled-components#7]
Hi! I just found this repo and after trying it out I noticed that the formatting of some styles using ``calc()`` are invalid. For example the generated media query styles used by the Container element looks like this:
```
{
  max-width: calc(1344px + (2 * 64px)) - (2 * 64px);
  ...
}
```

instead it should be formatted like this (the calc-function needs to wrap all calculations):
```
{
  max-width: calc(1344px + (2 * 64px) - (2 * 64px));
}
```

**Comment by goras** [comment]
Hello again! I don't know if any work has been done on this, but I just looked at the code and it's now a bit different from what I posted above. However, for me at least the issue still persists and I'm not able to use ``<Containers />`` properly.

In v1.0.1, styles for Container width renders as below and Chrome for instance, ignores the styles and gives the error ``Invalid property value``
```
@media screen and (min-width:960px){
  .jZgKqB{
    max-width:960px - (2 * 64px);
    width:960px - (2 * 64px);
  }
  ...
}
...
```

As far as I know it should instead render like below to be fully functional:
```
@media screen and (min-width:960px){
  .jZgKqB{
    max-width:calc(960px - (2 * 64px));
    width:calc(960px - (2 * 64px));
  }
  ...
}
...
```
